static/src/js/containers/AuthCallbackContainer/AuthCallbackContainer.js

```diff
@@ -39,42 +42,40 @@ export const AuthCallbackContainer = ({
       redirect = '/'
     } = params
 
-    // Verify that the redirect params are real URLs
-    try {
-      let redirectUrl
-      if (eddRedirect) redirectUrl = new URL(eddRedirect)
-      if (redirect && redirect !== '/') redirectUrl = new URL(redirect)
-
-      if (
-        redirectUrl
-        && redirectUrl.protocol !== 'http:'
-        && redirectUrl.protocol !== 'https:'
-        && redirectUrl.protocol !== 'earthdata-download:'
-      ) {
-        // The redirectUrl is not a valid protocol
-        console.log('The redirectUrl is not a valid protocol')
-        window.location.replace('/')
+    let eddRedirectUrl = eddRedirect
+    if (redirect.includes('earthdata-download')) {
```

Comment:
It might be better to do an exact match on the url just in case we use "earthdata-download" in a url sometime down the line. Seems safer to be a little more specific.

Reply:
One possibility is moving the test on line 53 with `startsWith` up here, which would mean only valid EDD redirects get processed by the following `if` block

Reply:
@rushgeo We could check if it starts with up there for `redirect`  but, we still need to have a `startsWith` check in case the `eddRedirectUrl` has been initialized to the value of the `eddRedirect`. I used `includes` because that was previous logic.